doc/interpolations.md

```diff
@@ -44,26 +44,30 @@ import inox._
 import inox.trees._
 import inox.trees.interpolator._
 
-implicit val symbols: trees.Symbols = trees.NoSymbols
+implicit val symbols: inox.trees.Symbols = inox.trees.NoSymbols
 ```
 
 Once imported, it is possible to build Inox types and expressions using a friendlier syntax:
 
 ```scala
-scala> val tpe = t"Boolean"
-tpe: inox.trees.interpolator.trees.Type = Boolean
-
-scala> val expr = e"1 + 1 == 2"
-expr: inox.trees.interpolator.trees.Expr = 1 + 1 == 2
+val tpe = t"Boolean"
+// tpe: Type = Boolean
+val expr = e"1 + 1 == 2"
+// expr: Expr = Equals(
```

Comment:
Do you know why pretty-printing is broken for expressions now?

Reply:
Hmm no that's strange

Reply:
It seems like we can use something like `mdoc:to-string` to use `toString()` instead of the weird custom printer: https://github.com/scalameta/mdoc/pull/148/files

Reply:
Thanks for looking this up! Should be good now